scripts/postlink/run.js

```diff
@@ -1,2 +1,10 @@
-require("./ios/postlink");
-require("./android/postlink");
\ No newline at end of file
+var postlinks = [
+    require("./ios/postlink"),
+    require("./android/postlink")
+];
+
+//run them sequentially
+var result = postlinks.reduce((p, fn) => p.then(fn), Promise.resolve());
```

Comment:
Do we really need result variable here?
Would it probably better to do the following:
```js
postlinks.reduce((p, fn) => p.then(fn), Promise.resolve())
    .catch((err) => {
        console.error(err.message);
    });
```

Reply:
Not at all, I'll simplify it as you suggested, @sergey-akhalkov, thx!